src/style.css

```diff
@@ -1,3 +1,81 @@
 body {
   margin: 0;
+  display: flex;
 }
+
+html {
+  margin: 0;
+  padding: 0;
+}
+
+header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  height: 60px;
+  flex-shrink: 0;
+  background: #FFF;
+  box-shadow:  2px 4px  rgba(0, 0, 0, 0.05);
+
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  flex-shrink: 0;
+  padding: 50px 10px;
+}
+
+
+
+ .nav__list {
+  list-style-type: none;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.nav__item:first-child {
+  margin-right: 0;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+}
+
+.link {
+  color: #000;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: normal;
+  text-decoration: none;
+  margin-right:  20px;
+  height: 15px;
+
+}
+
+.link:hover {
+  color:#00ACDC;
+}
+
+.is-active {
+  position: relative;
+  color: #00ACDC;
```

Comment:
Color `#00ACDC` should be stored in a variable in accordance with the task requirements:
![image](https://github.com/mate-academy/layout_moyo-header/assets/63243136/6b39d208-ccec-47fe-ab4d-79a286a8674e)